### PR TITLE
1.0.10: Arm32/Arm64 base image update

### PR DESCRIPTION
**Pull request by and-rewsmith**
Updating base images for security vulnerability

### CODE REVIEW

File: edge-agent/docker/linux/arm32v7/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.5.3-linux-arm32v7
```
Comment (and-rewsmith) [inline edge-agent/docker/linux/arm32v7/Dockerfile:1]: Bumping by 2 here to keep versions consistent

File: edge-agent/docker/linux/arm32v7/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+ARG base_tag=1.0.5.4-linux-arm32v7
```
Comment (yophilav) [inline edge-agent/docker/linux/arm32v7/Dockerfile:1]: The following images are not being updated correctly: 
```
MetricsValidator
ModuleRestarter
NumberLogger
Relayer
TestResultCoordinator
TwinTester
```

**Review by yophilav — CHANGES_REQUESTED** [review]

File: edge-agent/docker/linux/arm32v7/Dockerfile (hunk 2 above)
Comment (and-rewsmith) [reply edge-agent/docker/linux/arm32v7/Dockerfile:1]: Good catch I only updated the modules where the base images lived.

Comment (yophilav) [reply edge-agent/docker/linux/arm32v7/Dockerfile:1]: After you do the changes, could you please rebuild the whole thing?

Comment (and-rewsmith) [reply edge-agent/docker/linux/arm32v7/Dockerfile:1]: I would expect the list to be more similar to this from the master PR:
```
iotedge-diagnostics-dotnet
MetricsCollector
NetworkController
CloudToDeviceMessageTester
DeploymentTester
DirectMethodReceiver
DirectMethodSender
EdgeHubRestartTester
load-gen
MetricsValidator
ModuleRestarter
NumberLogger
Relayer
TemperatureFilter
TestResultCoordinator
TwinTester
```

Comment (yophilav) [reply edge-agent/docker/linux/arm32v7/Dockerfile:1]: Why is `NumberLogger` still using `1.0.6.2` ?

**Review by yophilav — CHANGES_REQUESTED** [review]

File: edge-agent/docker/linux/arm32v7/Dockerfile (hunk 2 above)
Comment (and-rewsmith) [reply edge-agent/docker/linux/arm32v7/Dockerfile:1]: It was probably an error missed in the last update. I will bump

**Review by yophilav — APPROVED** [review]

**Pull request merged**